calico/_config.yml

```diff
@@ -63,6 +63,11 @@ breadcrumbs:
 # Set to 'true' to show warning that points users to latest version of each page.
 is_latest: false
 
+# Flag that controls whether 'calicoctl' paragraphs and links are displayed in /Resource definitions. 
+# Allows CC docs to share Resource Definitions content with CE docs. 
+# Set to 'false' for Calico Cloud. Set to 'true' for Calico Enterprise and OS.
+include_calicoctl_resource: true
```

Comment:
I thought we agreed to just remove the calicoctl references altogether? I don't want to have to maintain yet another template parameter for this, I think it will just make maintenance harder over time.

The options are either to copy the content and remove the lines that aren't needed on the other end, or remove the lines here. I'm OK with the latter as a shortcut, even though the former is really the "correct" way to do it.

I'm not really OK with a new variable that we need to remember to wrap around every calicoctl mention in these docs.

Reply:
I thought we agree that I could not remove references for OS. Since discussion with Turk, I had another round with Karthik about it and we settled on what you see here. For CE, the idea was to add a blurb to https://projectcalico.docs.tigera.io/reference/calicoctl/overview to cover usage of calicoctl. If you are ok with removing it completely from OS, that is a better solution. Are you ok with that?